docs/reference/query-languages/esql/esql-commands.md

```diff
@@ -31,6 +31,7 @@ An {{esql}} source command produces a table, typically with data from {{es}}. An
 
 {{esql}} supports these processing commands:
 
+* [`CHANGE_POINT`](#esql-change-point)
```

Comment:
Should this have the `[preview]` prefix? I see you have the preview warning below, so I think perhaps the preview prefix should be here too, like it is for LOOKUP JOIN and MV_EXPAND.